### ISSUE TITLE
Pressure buffer file issue

### ISSUE THREAD

**Issue by lungd** [openworm/sibernetic#160]
What particles are supposed to be written to the pressure file?
The file should only contain particles belonging to the body of the worm, right?
If that is the case, I think I found a bug.

There should be 2289 elastic matter particles belonging to the worm using 'worm_crawl_half_resolution' config?
```
$ cat configuration/worm_crawl_half_resolution | grep -P ^"[0-9]+\.[0-9]+\s[0-9]+\.[0-9]+\s[0-9]+\.[0-9]+\s2\.[1-2]" | wc -l
2289
```
```
$ ./Release/Sibernetic -f worm_crawl_half_resolution -l_to -no_g timelimit=0.001
$ head buffers/pressure_buffer.txt
[Iteration 0]
Particle:	0
	Position:	4.19954	2.42451	4.19958	2.3
	Pressure:	0
Particle:	1
	Position:	4.21678	3.95076	4.21895	2.3
	Pressure:	0
Particle:	2
	Position:	4.56932	4.51407	4.54354	2.3
	Pressure:	0
```
There are particles with the type, e.g., '2.3' what is interpreted as an agar particle?
If I am right, only particles with '2.1 <= type < 2.3' should be inside the file.

Every iteration, 1425 particles get written to the file (should be 2289?)
```
$ cat buffers/pressure_buffer.txt | grep ^Particle: | wc -l
1425
```

**Comment by lungd** [comment]
BTW, there is one more particle using worm_swim_half_resolution?
```
$ cat configuration/worm_swim_half_resolution | grep -P ^"[0-9]+\.[0-9]+\s[0-9]+\.[0-9]+\s[0-9]+\.[0-9]+\s2\.[1-2]" | wc -l
2290
```

**Comment by skhayrulin** [comment]
@lungd Hi, I've actually supposed to store information about pressure only for particles which is located on the surface of body worm

**Comment by lungd** [comment]
@skhayrulin I understand now why there should be less than 2289 particles inside the pressure file.
But I am still not sure about the type of the particles.
Is the first particle located on the surface of the worm? (`Particle:	0 Position:	4.19954	2.42451	4.19958	2.3`)

**Comment by lungd** [comment]
@skhayrulin the question is, why there are particles of type=2.3 inside the file.

As I understand it, only a subset of particles of type > 2 and type < 2.3 should get saved.
So, I still think the content of the pressure file is not correct, or at least not completely as intended.

**Comment by skhayrulin** [comment]
@lungd Yep you're right it's the only subset of elastic particles but as far as I know, this indices needed @a-palyanov for manipulating the viscosity of liquid near the worm shell it's better to as him. From the other hand I don't use this I've precalculate set of shell particles from a list of membranes and put information about a particle inside the file as is.
